src/lib/export/exporters/tabby.ts

```diff
@@ -0,0 +1,36 @@
+import { toQueryString, type Recipe } from "$lib/ingredients";
+import { prepare } from "$lib/cereals";
+
+export function toTabby(recipe: Recipe): string {
+  const queryString = toQueryString(recipe);
+  const cereals = prepare(recipe);
+
+  return `# Copy the configuration below and add it under
+# terminal.customColorSchemes in the appropriate config file:
```

Comment:
I think the user experience isn't quite perfect here yet. When I installed tabby on my system, the `~/.config/tabby/config.yaml` file was empty. That way there was really no structure I could drop this scheme into.

I recommend we add the full yaml path (i.e. including the `terminal` and `customColorSchemes` sections in the generated config, so that it looks like this:

```yaml
terminal:
  customColorSchemes:
    - name: Root Loops
      foreground: '#000000'
      background: ...
```